src/Tools/dotnet-monitor/CollectionRules/ICollectionRuleAction.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Would it make sense for this to also go in the new Actions directory where we put `ExecuteAction.cs`? I realized I never clarified where this should go (or if it should just stay put).

Reply:
I think moving it to the Actions directory makes sense.